lhotse/__init__.py

```diff
@@ -33,7 +33,7 @@
     # NOTE: REMEMBER TO UPDATE THE ACTUAL VERSION IN setup.py WHEN RELEASING #
     # !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! #
     # Use a default placeholder when the version is unavailable...
-    __version__ = "1.3.0.dev+missing.version.file"
+    __version__ = "1.3.0+missing.version.file"
```

Comment:
After it is released, should the master contains`1.4.0.dev` since every later commit is not included in `1.3`?

Reply:
Yeah, I usually push the bump to next dev version directly to master.. I should remember to do this as a part of release PR.